cdk/autopilot/mapping/identifyMapping.yaml

```diff
@@ -0,0 +1,39 @@
+- destKey: Email
+  sourceKeys:
+  - traits.email
+  - context.traits.email
+  required: false
+- destKey: FirstName
```

Comment:
Please add at least 1 line gap b/w mapping sections for improving the readability.

Reply:
Sure